contributor/de/04-mechanics-of-contributing-article-de.asciidoc

```diff
@@ -0,0 +1,158 @@
+== Mechanik des Beitragens
+
+Sind Sie bereit, einen Beitrag zu Projekten/Repos anderer Teams zu leisten?
+Freuen Sie sich darauf, Ihre Blockaden nicht durch Management-Eskalation, sondern durch Zusammenarbeit abzubauen?
+In diesem Abschnitt finden Sie praktische Ratschläge und Hinweise, was Sie bei einem InnerSource-Beitrag beachten müssen. Er ermöglicht es Ihnen und dem Gastgeberteam, eine möglichst angenehme Erfahrung zu machen, die die Grundlage für weitere Beiträge und eine gute Zusammenarbeit bildet.
+
+Dieser Artikel ist in drei Schritte unterteilt, die Sie wahrscheinlich erleben werden
+
+* <<vorbereitungen-für-die-arbeit,Einholen des Beitragsangebots und Vorbereitung auf die Arbeit>>
+* <<erstellen-des-pull-requests,Den Beitrag tatsächlich erstellen>>
+* <<einreichen-des-pull-requests,Polieren und Verpacken des Geschenks und Überreichen an das Gastgeberteam>>.
```

Comment:
This looks like there used to be markdown links that are broken. Probably needs to be fixed.